org.eclipse.scanning.sequencer/src/org/eclipse/scanning/sequencer/AcquisitionDevice.java

```diff
@@ -550,22 +552,22 @@ public void resume() throws ScanningException {
 	@Override
 	public void positionPerformed(PositionEvent evt) throws ScanningException {
 		IPosition position = evt.getPosition();
-		try {
```

Comment:
Does this work with non-malcolm devices? There may not be an inner position, does that matter?